flink-cdc-connect/flink-cdc-pipeline-connectors/flink-connector-test-cdc/src/main/java/com/ververica/cdc/connectors/test/sink/ValuesDataSinkFactory.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.test.sink;
+
+import com.ververica.cdc.common.configuration.ConfigOption;
+import com.ververica.cdc.common.factories.DataSinkFactory;
+import com.ververica.cdc.common.sink.DataSink;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/** A dummy {@link DataSinkFactory} for testing. */
+public class ValuesDataSinkFactory implements DataSinkFactory {
+
+    @Override
+    public DataSink createDataSink(Context context) {
+        int flushSize = context.getConfiguration().get(ValuesDataSinkOptions.FLUSH_SIZE);
+        return new ValuesDataSink(flushSize);
+    }
+
+    @Override
+    public String identifier() {
+        return "test-sink";
```

Comment:
Could we call this sink `values` simply? My proposed name will be more aligned with `ValuesDataSink`.

Reply:
And we can use same identifier in ValuesDataSinkFactory  and ValuesDataSourceFactory